.github/workflows/main.yml

```
@@ -38,11 +38,11 @@ jobs:
       - name: Download files
         if: steps.cache.outputs.cache-hit != 'true'
         run: |
-          curl -JLOs "https://jenkins.pmmp.io/job/PHP-8.0-Aggregate/lastSuccessfulBuild/artifact/PHP-8.0-Windows-x64-PM4.zip" &
+          curl -JLOs "https://github.com/pmmp/PHP-Binaries/releases/download/php-8.0-latest/PHP-Windows-x64-PM4.zip" &
```

```suggestion
          curl -JLOs "https://github.com/pmmp/PHP-Binaries/releases/latest/download/PHP-Windows-x64-PM4.zip" &
```
tagの指定でも良いと思いますが、おそらくlatestから取ってくるほうが良いのではないでしょうか？